docs/modding/workshop/workshopoverview.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Portal 2: Community Edition Workshop"
```

Comment:
Please use game neutral titles, if features (eg. the workshop) are only supported in some games, use [feature flags](https://wiki.stratasource.org/contribute/basics/feature-flags).

Reply:
Alright, how would I go about adding a new feature flag? The workshop this documentation is referring to is for P2CE only, but the wiki doesn't have a - WORKSHOP feature flag.

Reply:
Yeah, I'd just call this page "Workshop" instead.

Reply:
Just normalized the feature flags, the one for workshop is `USE_UGC`.

Reply:
Alright, done
Ive also added - USE_UGC feature flag to both pages